src/frontend/components/SVGIcons/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+import { normalizeColor } from 'grommet/utils';
+
+import { theme } from 'utils/theme/theme';
+
+const StyledSVG = styled.svg<SvgProps>`
+  path {
+    fill: ${(props) => normalizeColor(props.baseColor, theme)};
+  }
+
+  :hover {
+    path {
+      fill: ${(props) => normalizeColor(props.hoverColor, theme)};
+    }
+  }
+`;
+
+export interface SvgProps {
+  baseColor: string;
+  hoverColor: string;
+}
+
+interface SVGIconProps extends SvgProps {
+  title: string;
+  width: string;
+  height: string;
```

Comment:
I suggest you to move these 3 proprs in `SvgProps`. You are adding them in all `SVG` components using this one.